microsetta_public_api/api/metadata.py

```diff
@@ -12,8 +12,22 @@ def categories():
     return jsonify(repo.categories), 200
 
 
+def categories_alt(dataset):
+    repo = _get_repo_alt(dataset)
+    return jsonify(repo.categories), 200
+
+
 def category_values(category):
```

Comment:
Can the non-dataset specific pieces be removed?

Reply:
Or would that make sense in a separate PR?

Reply:
Removing the non-dataset specific pieces would be a huge PR and result in a lot of headaches. Can do, but I would estimate that not being done for 3-4 weeks from now. So if we want to work with these pieces in the mean time, it would be better to do it as a separate PR.